Core/scripts/dirac-install.py

```diff
@@ -2559,7 +2560,8 @@ def createSymbolicLink():
   else:
     logNOTICE("Skipping installing DIRAC")
 
-  if cliParams.diracOS:
+  # we install with DIRACOS from v7rX DIRAC release
+  if cliParams.diracOS or int(releaseConfig.prjRelCFG['DIRAC'].keys()[0][1]) > 6:
```

Comment:
If you do it like this, there is no way to turn off dirac-os for v7, is there?

Reply:
Correct.

Reply:
At the same there's no "--externals" switch.